### ISSUE TITLE
M1 Compatibility

### ISSUE THREAD

**Issue by humandecoded** [tracelabs/tlosint-vm#3]
Currently, the build process is only tested on Intel arch. Can it work if we run it on M1 macs?

**Comment by mmuyskens** [comment]
One potential issue is that debos appears to either not support ARM64 arch, or at minimum, is a bit hackey to get running.

https://github.com/go-debos/debos/issues/386

**Comment by humandecoded** [comment]
Yeah. I seem to remember that from the last time I looked in to this. Might be worth looking at the Kali builds and see how they are doing it...(a few moments later)...Looks like they only offer VM builds for Intel but offer installer ISOs for M1 mac. So may just have to update our current M1 build repo to mirror the functionality of this one. Shouldn't be too hard once we get this one where we want it

**Comment by humandecoded** [comment]
> They have Kali for ARM, maybe we can have another repo just for ARM? we will customize our script accordingly as well as the build processes

I think is what we'll end up doing. We've been able to do arm builds previously 
https://github.com/tracelabs/Trace-Labs-VM-M1-Mac-Build
but don't think we're able to do that with this workflow

**Comment by humandecoded** [comment]
M1 builds are currently live over at https://github.com/tracelabs/Trace-Labs-VM-M1-Mac-Build
The best option we have now is to generate an ISO and install from there. We could go through the process of exporting a VMware compatible image but that would put us right back where we were with the original repo

**Comment by UltraSunshine** [comment]
This is in this PR: https://github.com/tracelabs/tlosint-vm/pull/175

But I need to retarget it at main. Will do soonly

**Comment by UltraSunshine** [comment]
This is staged in https://github.com/tracelabs/tlosint-vm/pull/182 but I need an adult to merge it

**Comment by Tayzar02** [comment]
Closed by #189
